workflow-report/action.yml

```diff
@@ -0,0 +1,73 @@
+name: 'Send VK Teams notifications about failed job'
+description: 'This action create a message about failed job and send it to VK Teams chat'
+
+inputs:
+  api-url:
+    description: 'Bot API URL (ICQ/Myteam)'
+    required: false
+    default: 'https://api.internal.myteam.mail.ru/bot/v1/'
+  bot-token:
+    description: 'Bot token'
+    required: true
+  chat-id:
+    description: 'Notification chat ID (or stamp from chat URL)'
+    required: true
+  workflow-steps:
+    description: 'Get steps to show it in message'
+    required: false
+
+runs:
+  using: "composite"
```

Comment:
Please, use single quotes or don't use at all.

Reply:
Done

Reply:
Are you sure?

Reply:
Strange. Yes for now